releasenotes-builder/src/test/resources/com/github/checkstyle/githubPageAll.txt

```diff
@@ -2,16 +2,16 @@ Checkstyle 1.0.0 - https://checkstyle.org/releasenotes.html#Release_1.0.0
 
 Breaking backward compatibility:
 
-  #-1 - Title 1
+  #1 - Title 1
```

Comment:
This is a bug in the prior tests. As we found out when we fail to find issue, it is exception. If it is not an issue commit, it is auto added as miscellaneous and goes under `Other Changes`.
https://github.com/checkstyle/contribution/blob/654a97ee7883f0e46cce0ff11bfb8a6958e3c970/releasenotes-builder/src/main/java/com/github/checkstyle/github/NotesBuilder.java#L175

Reply:
Noticed that as well when writing tests.